src/modules/ekf2/ekf2_params.c

```diff
@@ -644,6 +644,21 @@ PARAM_DEFINE_INT32(EKF2_AID_MASK, 1);
  */
 PARAM_DEFINE_INT32(EKF2_HGT_MODE, 0);
 
+/**
+ * Integer bitmask controlling fusion sources of the terrain estimator
+ *
+ * Set bits in the following positions to enable:
+ * 0 : Set to true to use range finder data if available
```

Comment:
@bresch You could almost remove this as the description of the bits below is very clear.
Just a comment, don't need to change it.